devito/nodes.py

```diff
@@ -415,6 +415,20 @@ def name(self):
         return self._name
 
 
+class Denormals(Block):
+
+    """Macros to make sure denormal numbers are flushed in hardware."""
+
+    def __init__(self, header=None, body=None, footer=None):
+        b = [Element(c.Comment('Flush denormal numbers to zero in hardware')),
+             Element(c.Statement('_MM_SET_DENORMALS_ZERO_MODE(_MM_DENORMALS_ZERO_ON)')),
+             Element(c.Statement('_MM_SET_FLUSH_ZERO_MODE(_MM_FLUSH_ZERO_ON)'))]
```

Comment:
Can we just call these functions directly from python (`ctype`), instead of adding them to the code every time it is generated?

Not sure which would be considered better.